src/forms/PayoutDetailsForm/PayoutDetailsAddress.js

```diff
@@ -148,37 +134,62 @@ const PayoutDetailsAddress = props => {
           />
         ) : null}
       </div>
-      {showState ? (
-        <FieldTextInput
+      {showProvinceCA ? (
+        <FieldSelect
+          id={`${fieldId}.province`}
+          name={`${fieldId}.province`}
+          disabled={disabled}
+          className={css.selectCountry}
+          autoComplete="address-level1"
+          label={provinceLabel}
+          validate={provinceRequired}
+        >
+          <option disabled value="">
+            {provincePlaceholder}
+          </option>
+          {CA_PROVINCES.map(p => (
```

Comment:
Am I correct that this array is the only difference between these different selects?
Howabout checking 

```js
const states = showStateUS 
  ? US_STATES 
  : showProvinceCA 
  ? CA_PROVINCES
  :  showStateAU
  ? AU_STATES
  : [];
```
And then creating one select that uses `states` array as the source of data?